order-service/src/main/resources/db/migration/V1__Init_DB.sql

```diff
@@ -1,10 +1,23 @@
+-- Create schema for the application
+CREATE SCHEMA IF NOT EXISTS kss;
+
 -- Create sequences for ID columns
+CREATE SEQUENCE customer_id_seq;
 CREATE SEQUENCE item_id_seq;
 CREATE SEQUENCE item_category_id_seq;
 CREATE SEQUENCE k_order_id_seq;
 CREATE SEQUENCE order_line_item_id_seq;
 
 -- Create tables for entities
+CREATE TABLE customer
+(
+    id         BIGINT DEFAULT nextval('customer_id_seq') PRIMARY KEY,
+    name       VARCHAR(255),
+    email      VARCHAR(255)
+--     created_at TIMESTAMP,
+--     updated_at TIMESTAMP
```

Comment:
I think i would by nice to add to BaseEntity what do you think ?

Reply:
Agree. Or, if we need more flexibility, we could have a TimestampedBaseEntity derived from BasedEntity. For now, I'd move it directly to the BaseEntity. I'd like to do this in a separate PR, if that's ok? Thank you.

Reply:
I've added this now.